### ISSUE TITLE
image preprocessing for training

### ISSUE THREAD

**Issue by keighrim** [clamsproject/app-swt-detection#2]
For the first batches of training, we plan to use video frames extracted from AAPB videos. However, there's concerns around how to "normalize" different image size and aspect ratios from videos from different decades. This thread is to discuss how we implement the normalizing strategies.

**Comment by keighrim** [comment]
We decided to go only with 4:3 videos (before HD broadcasting era, circa early 2000s) in the early rounds of annotation.

**Comment by keighrim** [comment]
Note that as long as we are using the pre-trained backbone weights, those weights in `torch-vision` package come with their own preprocessing code. That is, we can add some additional preprocessing based on some domain knowledge before the torch-shipped preprocessing. However, as of now we don't see a lot of needs for doing so.
